ambari-common/src/main/python/resource_management/libraries/execution_command/execution_command.py

```diff
@@ -77,12 +80,18 @@ def get_value(self, query_string, default_value=None):
   def get_module_configs(self):
     return self.__module_configs
 
+  def get_stack_settings(self):
+    return self.__stack_settings
+
+  def get_cluster_settings(self):
+    return self.__cluster_settings
+
   def get_module_name(self):
     """
     Retrieve service type from command.json, eg. 'zookeeper', 'hdfs'
     :return: service type
     """
-    return self.__get_value("serviceType")
+    return self.__get_value("serviceName")
```

Comment:
How will this work with custom service names, eg. `"ZK1"`?

Reply:
Do we have serviceType in command.json? I cannot find it any more, could you give me a latest command.json which includes this entry? Thanks.